todo_app/tests_e2e/test_app.py

```diff
@@ -2,20 +2,14 @@
 import os 
 
 from todo_app.app import create_app
-from todo_app.trello.trello_api import TrelloAPI
-from todo_app.trello.trello_api import TrelloBase
 from dotenv import load_dotenv, find_dotenv
 from selenium import webdriver
 from selenium.webdriver.chrome.options import Options
 from threading import Thread
 
-
 @pytest.fixture(scope='module')
 def test_app():    
 
-    # Create the new board and update the board id environment variable
-    os.environ['TRELLO_BOARD_ID']  = TrelloAPI.create_trello_board("E2E test Board")
-
```

Comment:
Nice, but it would be good to also override the database name to a test one so we don't accidentally interfere with the live db here